package.json

```diff
@@ -7,6 +7,7 @@
     "@slack/socket-mode": "^1.2.0",
     "@slack/web-api": "^6.7.1",
     "axios": "^0.27.2",
+    "bootstrap-icons": "^1.8.1",
```

Comment:
remove everything related to bootstrap icons, its a very big package and for a very small thing we should not install a big package.

Reply:
for icon can i use font awesome instead of bootstrap.